kong.conf.default

```diff
@@ -829,6 +829,15 @@
                                  # for a short period of time after Routes and
                                  # Services updates.
 
+#router_update_frequency = 1     # Defines how often the router changes are
+                                 # checked with a background job. When a change
+                                 # is detected, a new router will be built. By
+                                 # default we check for changes every second.
+                                 # Raising this value may allow more requests
```

Comment:
Instead of "allow more requests to flow", how about "Raising this value will decrease the load on database servers and results in less jitter in proxy latency, with downside of longer waits for router update to converge.".